libraries/heft-config-file/src/ConfigurationFile.ts

```diff
@@ -767,20 +767,20 @@ export class ConfigurationFile<TConfigurationFile> {
           case InheritanceType.merge: {
             if (parentPropertyValue === null || propertyValue === null) {
               throw new Error(
-                `Issue in processing configuration file property "${propertyName}". ` +
+                `Issue in processing configuration file property "${String(propertyName)}". ` +
                   `Null values cannot be used when the inheritance type is set as "${InheritanceType.merge}"`
               );
             } else if (
               (propertyValue && typeof propertyValue !== 'object') ||
               (parentPropertyValue && typeof parentPropertyValue !== 'object')
             ) {
               throw new Error(
-                `Issue in processing configuration file property "${propertyName}". ` +
+                `Issue in processing configuration file property "${String(propertyName)}". ` +
                   `Primitive types cannot be provided when the inheritance type is set as "${InheritanceType.merge}"`
               );
             } else if (Array.isArray(propertyValue) || Array.isArray(parentPropertyValue)) {
               throw new Error(
-                `Issue in processing configuration file property "${propertyName}". ` +
+                `Issue in processing configuration file property "${String(propertyName)}". ` +
```

Comment:
What is `"${String(propertyName)}"`?

Reply:
TS gives a warning that `propertyName` might be a `symbol`, and using that in a template string throws. Changed this to `.toString()`